content/system76-driver.md

```diff
@@ -97,19 +97,20 @@ makepkg -si
 
 **NOTE** that by default <u>Paru</u> uses vim standards so when you see a ":" press the <kbd>q</kbd> key to continue. You may also need to import some public keys by using the <kbd>y</kbd> key.
 
-Now the <u>System76 Driver</u> can be cloned, built and installed using these commands:
+Next, build the <u>System76 Driver</u> with <u>Paru</u>
+**NOTE** choose the first software option listed after running the <u>Paru</u> command.
```

Comment:
We have two all-caps bold **NOTES** within three paragraphs of each other. This is an instruction, can it not just be stated plainly?

If you did keep the **NOTE**, it needs a colon after it in this context. (It looks like that earlier one doesn't have a colon because it's part of the sentence, which I would not have agreed to, but that doesn't work here.)